WpfTest/ViewModel/MainPageViewModel.cs

```diff
@@ -0,0 +1,85 @@
+    using System.Collections.ObjectModel;
+using System.Windows.Controls;
+using System.Windows.Input;
+using WpfTest.Commands;
+using WpfTest.View;
+using WpfTest.Model;
+
+namespace WpfTest.ViewModel;
+
+public class MainPageViewModel : BaseViewModel
+    {
+        public ObservableCollection<Project> Projects { get; set; }
+
+        private Project _selectedProject;
+        private Page? _selectedProjectPage;
+        
+        public ICommand AddCommand { get; }
+        
+        public Project SelectedProject
+        {
+            get => _selectedProject;
+            set
+            {
+                _selectedProject = value;
+                OnPropertyChanged(nameof(SelectedProject));
+                NavigateToProjectPage(value);
+                
+            }
+        }
+        
+        public Page? SelectedProjectPage
+        {
+            get => _selectedProjectPage;
+            set
+            {
+                _selectedProjectPage = value;
+                OnPropertyChanged(nameof(SelectedProjectPage));
+            }
+        }
+        
+        public MainPageViewModel()
+        {
+            Projects = ProjectManager.GetProjects();
+            AddCommand = new RelayCommand(Add, CanAdd);
+        }
+
+        private bool CanAdd(object arg)
+        {
+            return true;
+        }
+
+        private void Add(object obj)
+        {
+            CreateProjectWindow createProjectWindow = new CreateProjectWindow();
+            if (createProjectWindow.ShowDialog() == true)
```

Comment:
no need to check that variable is true or false, since your function returns boolean value, just make like this:

```c#
if (createProjectWindow.ShowDialog())
```